tests/test_core.py

```diff
@@ -783,6 +783,13 @@ def __init__(self):
         imageio.imwrite("foo.bmp", "asd")
 
 
+def test_imwrite_symbol_name():
```

Comment:
```suggestion
def test_imwrite_symbol_name():
    # this is a regression test for https://github.com/imageio/imageio/issues/674
```

added xref to keep track of things.